site/content/en/docs/api_sdk/sdk/lowlevel-api.md

```diff
@@ -641,3 +642,70 @@ with ApiClient(
     pprint(files) # ['sub/image_1.jpg', 'image_2.jpg']
 
 ```
+
+### Sending custom requests
+
+In some cases, you might need to send a custom request while using the `ApiClient`.
+Typically, it's desirable to make this request using the same configuration
+and authentication parameters as regular requests sent via the `ApiClient` instance.
+
+One particularly useful example for this is dataset export. When exporting a dataset,
+you receive a download URL from the server after the file is prepared
+(see example in [receiving data](#receiving-data)). This URL requires authentication,
+but it can't be made via the regular `ApiClient` endpoint methods.
+
+There are several options to make such a custom request via an `ApiClient` instance:
+- use `api_client.call_api()`
+- use `api_client.request()` or `api_client.rest_client` methods
+  together with `api_client.update_params_for_auth()`
+
+{{< tabpane text=true >}}
+
+{{% tab header="Using api_client.call_api()" %}}
+
+This option provides higher-level interface and allows better integration with other
+`ApiClient`-related interfaces, such as `Endpoint`.
+
+```python
+from urllib.parse import parse_qsl, urlparse
+
+with ApiClient(...) as api_client:
+    parsed_url = urlparse("<custom URL>")
+    query_params = parse_qsl(parsed_url.query)
+    _, response = api_client.call_api(
+        parsed_url.path,
+        method="GET",
+        query_params=query_params,
+        _parse_response=False,
+    )
+
+    # process response.data ...
+```
+
+{{% /tab %}}
+
+{{% tab header="Using api_client.request()" %}}
+
+This option provides a lower-level interface and allows more customization. It can be useful if
+you need an API similar to what the famous `requests` library provides or want to send the request
```

Comment:
This doesn't really make sense to me. If someone needs an API like `requests`, couldn't they use `requests` itself? Why do we even need to offer this?

Reply:
`rest_client` has some configuration - e.g. connection pool, timeouts etc. If the user wants to make a customized request, but use the same configuration like other `ApiClient` methods do, these methods can be used.